test/logstream/interface.go

```diff
@@ -31,7 +32,10 @@ import (
 
 // Canceler is the type of a function returned when a logstream is started to be
 // deferred so that the logstream can be stopped when the test is complete.
-type Canceler = logstreamv2.Canceler
+type (
+	Canceler = logstreamv2.Canceler
+	Callback = logstreamv2.Callback
```

Comment:
I guess we need to document `Callback` too here.

Reply:
Oh, I see.

Reply:
Pushed.